runtime/src/main/java/org/corfudb/runtime/collections/CorfuTable.java

```diff
@@ -267,26 +267,39 @@ public ProjectionFunction getProjectionFunction() {
         }
     }
 
-    /** The "main" map which contains the primary key-value mappings. */
-    private final Map<K,V> mainMap = new HashMap<>();
+    // The "main" map which contains the primary key-value mappings.
+    private final Map<K,V> mainMap;
     private Set<Index<K, V, ? extends Comparable>> indexSpec = new HashSet<>();
     private final Map<String, Map<Comparable, Map<K, V>>> secondaryIndexes = new HashMap<>();
 
     @Getter
     boolean indexGenerationFailed = false;
```

Comment:
private and final if possible

Reply:
CorfuTable has several things that are non-conformant. I explicitly refrained from addressing those in this PR. We should have a separate PR addressing those since there are so many.